configuration/README.md

```diff
@@ -36,36 +31,24 @@ type Example struct {
 ```
 
 The tag `levee:"source"` is built-in. Additional tags may be identified via explicit string literals (not regexps). The following example shows how the `levee:"source"` tag could be defined if it weren't built-in:
-```json
-{
-	"FieldTags": [
-		{
-			"Key": "levee",
-			"Val": "source"
-		}
-	]
-}
+```yaml
+FieldTags:
+- Key: levee
+  Val: source
 ```
 
 Sinks and sanitizers are identified via regexp according to package, method, and (optional) receiver name.
 
-```json
-{
-  "Sinks": [
-    {
-      "PackageRE": "<package path regexp>",
-      "ReceiverRE": "<type name regexp>",
-      "MethodRE": "<method name regexp>"
-    }
-  ],
-  "Sanitizers": [
-    {
-      "PackageRE": "<package path regexp>",
-      "ReceiverRE": "<type name regexp>",
-      "MethodRE": "<method name regexp>"
-    }
-  ]
-}
+```yaml
+Sinks:
+  - PackageRE: <package path regexp>,
```

Comment:
Extraneous comma detected.

Reply:
Those commas made me chase my tail in testing.  You'd think I'd have been trained to notice them.